src/control/vendor/github.com/daos-stack/go-spdk/spdk/src/nvme_control.c

```diff
@@ -498,7 +495,7 @@ nvme_format(char *ctrlr_pci_addr)
 	format.ms	= 0; // metadata transferred as part of a separate buffer
 	format.pi	= 0; // protection information is not enabled
 	format.pil	= 0; // protection information location N/A
-	format.ses	= 0; // no secure erase operation requested
+	format.ses	= 1; // no secure erase operation requested
```

Comment:
Should this be "User data erase" instead of "No secure erase operation requested"?  Would there be a case where we need to provide 0 as an argument or will this always be the default?

Reply:
that is how the parameter is described in SPDK. we will always want user data to be removed because this will include blobstores.

Reply:
Indeed, the comment needs be updated if the patch needs be refreshed.

Reply:
addressed in https://github.com/daos-stack/go-spdk/pull/7 will be pulled in next time vendor version is bumped in DAOS